Makefile

```diff
@@ -182,16 +182,11 @@ spectest.v1:
 spectest.v2:
 	@go test $$(go list ./... | grep $(spectest_v2_dir)) -timeout 120s
 
-golangci_lint_path := $(shell go env GOPATH)/bin/golangci-lint
-
-$(golangci_lint_path):
-	@go install $(golangci_lint)
-
 golangci_lint_goarch ?= $(shell go env GOARCH)
 
 .PHONY: lint
 lint: $(golangci_lint_path)
-	@GOARCH=$(golangci_lint_goarch) CGO_ENABLED=0 $(golangci_lint_path) run --timeout 5m
+	@GOARCH=$(golangci_lint_goarch) CGO_ENABLED=0 go run $(golangci_lint) run --timeout 5m
```

Comment:
I had some incompatible version installed. Using `go run` avoids interaction with whatever's already installed

Reply:
Reverted for now though since had bad interaction with CI